examples/src/main/kotlin/org/wfanet/examples/streaming/HelloStreamingClient.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 The Cross-Media Measurement Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wfanet.examples.streaming
+
+import io.grpc.ManagedChannel
+import io.grpc.ManagedChannelBuilder
+import java.io.Closeable
+import java.util.concurrent.TimeUnit
+import kotlinx.coroutines.flow.asFlow
+import kotlinx.coroutines.flow.collect
+import org.wfanet.examples.streaming.HelloStreamingServiceGrpcKt.HelloStreamingServiceCoroutineStub
+
+class HelloStreamingClient(private val channel: ManagedChannel) : Closeable {
+  private val stub = HelloStreamingServiceCoroutineStub(channel)
+
+  suspend fun helloStreaming(names: Array<String>) {
+    val requests = names.map { name ->
+      val request = HelloStreamingRequest.newBuilder().setName(name).build()
+      println("Sending: ${request.name}")
```

Comment:
Technically, this isn't quite true. If you want it to actually print when it's sending, use an `onEach` on the flow, or move the asFlow before the map:

Option 1:
```
val requests =
  names
    .map { name -> HelloStreamingRequest.newBuilder().setName(name).build() }
    .asFlow()
    .onEach { println("Sending: ${it.name}") }
```

Option 2:

```
val requests =  names.asFlow().map { name -> ... }
```

Reply:
Done